draft-pauly-httpbis-geoip-hint.md

```diff
@@ -24,36 +24,70 @@ author:
-
-This document defines a client hint that can be used to send a IP geolocation entry that
-maps to the client's IP address. This location can be used to influence server behavior,
-such as by causing the server to return responses relevant to the client's location.
-The format of the IP geolocation entry is the same as that defined for IP geolocation
-feeds in {{!GEOFEED=RFC8805}}.
-
-This header is intended to be used to provide rough geolocation hints to servers that do
-not already have accurate or authoritative mappings for the IP addresses of clients. This
-can be particularly useful for cases where IP geolocation mappings have changed recently,
-or a client is using a VPN or proxy that may not be commonly recognized by servers.
-
-The mechanism for how a client learns the IP geolocation mapping to send is beyond the
-scope of this document. {{?RFC9092}} defines some mechanisms for discovery, but clients
-can also have other mechanisms (such as coordinating with a VPN or proxy that is assigning
-the client a tunnelled or proxied address) to learn what hint to sent.
+HTTP Client Hints {{!RFC8942}} defines a convention for HTTP headers to
+communicate optional information from clients to servers as hints. This
+can be done conditionally based on if a server claims to support a
+particular hint. A server can request hints by listing them in the
+Accept-CH response header.
+
+This document defines a client hint that can be used to send a
+geolocation entry based on the client's determined location. This
+location can be used to influence server behavior, such as by causing
+the server to return responses relevant to the client's location. The
+format of the geolocation hint is the same as that defined for IP
+geolocation feeds in {{!GEOFEED=RFC8805}}. It only allows for
+coarse-level location specification.
+
+This header aims to provide rough geolocation hints to servers based on
+the client’s network location, shifting geolocation from a passive
+IP-based approach to an active client-controlled one. This not only
+allows the client to influence how their location is interpreted, but
+it also reduces the need for extensive IP address pools when clients
+mask their IP addresses through VPNs or proxies. Typically, VPN or proxy
+providers need to purchase egress IPs for each region to maintain
+accurate geolocation. With a client-provided location hint, the hint can
+minimize the number of IP addresses needed while still supporting
+location-specific content such as weather, local news, and search
+results. In addition, the hint reduces reliance on geo-IP
+feeds that often come with limitations such as outdated
+IP-to-location mappings and ongoing maintenance costs.
+
+The client MUST determine geolocation via a cooperating server
+that performs a geo-IP database lookup of the client's IP address.
+Geolocation MUST NOT be derived from GPS or any method that
+provides more specificity than the information obtainable from the
+client's IP address.
```

Comment:
One of the conventions in RFCs is to not duplicate normative language (MUST/SHOULD/etc). Since we already have the MUST in the Client Behavior section, I would use prose here instead

```suggestion
The client determines geolocation via a cooperating server
that performs a geo-IP database lookup of the client's IP address.
```